src/gateio_new_coins_announcements_bot/main.py

```diff
@@ -17,9 +20,11 @@
 from gateio_new_coins_announcements_bot.trade_client import get_last_price
 from gateio_new_coins_announcements_bot.trade_client import place_order
 
+init_logger()
```

Comment:
This should be moved into `<root>/main.py` or `main()`

Reply:
initializing the logger later (in main()) requires the lines afterwards to be moved as well since they use the LOG functions.
I could however move it into the initial call in `<root>/main.py` but after further thinking that could cause problems if someone still wanna call src/main.py directly (which is currently possible) and in general isn't that good because the main() function should really handle the setup itself.

In a later PR i will refactor the initial setup stuff into a separate function and move the `init_logger()` then as well.
I will then also remove the `if __name__ == "__main__":` statement since its redundant, you should only be able to run the main.py in the root directory